examples/video_understanding/configs/workers/video_qa.yml

```
@@ -0,0 +1,5 @@
+name: VideoQA
+llm: ${sub|gpt4o}
+output_parser: 
+  name: StrParser
+text_encoder: ${sub| text_encoder}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)